### PR TITLE
Allow getting the policy of a bucket

### PR DESCRIPTION
**Pull request by aleb**
The struct used to deserialize the response content is very limited, has only the `version` and `id` fields, but it's a good start nevertheless.

Threw in two other small commits, please review each commit separately. Thanks!

### CODE REVIEW

**Comment by aleb** [comment]
Is this of interest?

File: src/actions/mod.rs [hunk 1]
```diff
@@ -9,6 +9,7 @@ pub use self::delete_bucket::DeleteBucket;
 pub use self::delete_object::DeleteObject;
 #[cfg(feature = "full")]
 pub use self::delete_objects::{DeleteObjects, ObjectIdentifier};
+pub use self::get_bucket_policy::{GetBucketPolicy, GetBucketPolicyResponse};
```
Comment (paolobarbolini) [inline src/actions/mod.rs:12]: This needs to be gated on the `full` feature as it requires dependencies that are only available when it's enabled. You can check if it compiles afterwards with `cargo check --no-default-features`

File: src/actions/mod.rs [hunk 2]
```diff
@@ -31,6 +32,7 @@ mod delete_bucket;
 mod delete_object;
 #[cfg(feature = "full")]
 mod delete_objects;
+mod get_bucket_policy;
```
Comment (paolobarbolini) [inline src/actions/mod.rs:35]: Same here

File: src/actions/get_bucket_policy.rs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    pub fn new(bucket: &'a Bucket, credentials: Option<&'a Credentials>) -> Self {
+        Self {
+            bucket,
+            credentials,
+
+            query: Map::new(),
+            headers: Map::new(),
+        }
+    }
+    pub fn parse_response(s: &str) -> Result<GetBucketPolicyResponse, serde_json::Error> {
```
Comment (paolobarbolini) [inline src/actions/get_bucket_policy.rs:49]: Nit: there should be a separator line here

```suggestion
    }

    pub fn parse_response(s: &str) -> Result<GetBucketPolicyResponse, serde_json::Error> {
```

**Review by paolobarbolini — CHANGES_REQUESTED** [review]
We are interested in this feature getting merged. Me and @guerinoni work on this project when we have time to.

**Comment by aleb** [comment]
Done

**Review by paolobarbolini — APPROVED** [review]
LGTM. Thank you for the contribution!

**Pull request merged**